### ISSUE TITLE
Rerun EPU analysis 

### ISSUE THREAD

**Issue by khyde** [NOAA-EDAB/seaside#6]
Rerun the EPU analysis to see if the regions have changed and develop metrics to indicate "change"

**Comment by andybeet** [comment]
Did you mean to add this issue in this repo?

**Comment by khyde** [comment]
Yes, we had a discussion about it and Scott asked me to add it.  The idea is that we can discuss the best practices for rerunning the analysis and how to make it easier to do as needed.

**Comment by kimberly-bastille** [comment]
Mike/Rob to do something about this on 5/18/20.
